sdk/communication/communication-phone-numbers/src/models.ts

```diff
@@ -76,12 +77,18 @@ export interface SearchOperatorInformationOptions extends OperationOptions {
 /**
  * Additional options that can be passed to list SIP routes.
  */
-export interface ListSipRoutesOptions extends OperationOptions {}
+export interface ListSipRoutesOptions extends OperationOptions {
+  /** Sip configuration expand. Optional. */
+  expand?: ExpandEnum;
+}
 
 /**
  * Additional options that can be passed to list SIP trunks.
  */
-export interface ListSipTrunksOptions extends OperationOptions {}
+export interface ListSipTrunksOptions extends OperationOptions {
```

Comment:
Would it make more sense if SipRoutingGetOptions extends SipRoutingGetOptions instead?
That way, we propagate the expand?: ExpandEnum through inheritance and keep the code DRY.